lambdas/service/app.py

```diff
@@ -783,6 +780,21 @@ def validate_order(order: str):
         raise BRE(f'Unknown order `{order}`. Must be one of {supported_orders}')
 
 
+def validate_search_before(search_before: str):
+    validate_json_param('search_before', search_before)
+
+
+def validate_search_after(search_after: str):
+    validate_json_param('search_after', search_after)
+
+
+def validate_json_param(name: str, json_: str) -> MutableJSON:
```

Comment:
```suggestion
def validate_json_param(name: str, value: str) -> MutableJSON:
```

Underscore should be a last resort.